docs/guidelines.rst

```diff
@@ -43,9 +43,15 @@ Create a pull request
   This means that we can NOT accept pull requests from you unless this is done,
   so please don't put the code reviewers at risk and do it anyways.
 
-* Add a changelog entry in ``CHANGES.rst``.
-  Make it clear if it is a fix or a new feature.
+* Add a changelog entry as a `towncrier <https://github.com/hawkowl/towncrier>`_ news entry in the ``news`` directory in the repository root.
+  The entry should make clear what your change is about.
   When in doubt, don't worry about it.
```

Comment:
This old line was meant to follow after "Make it clear if it is a fix or a new feature." It can either be removed, or moved below your new line with feature/bugfix/breaking. A better text may then be:
"When in doubt, pick 'bugfix'."